src/applications/personalization/profile/hooks/useDirectDeposit.js

```diff
@@ -102,7 +102,11 @@ export const useDirectDeposit = () => {
   const onFormSubmit = useCallback(
     () => {
       const payload = {
-        paymentAccount: formData,
+        paymentAccount: {
+          accountType: formData.accountType,
+          accountNumber: formData.accountNumber,
+          routingNumber: formData.routingNumber,
+        },
```

Comment:
Here I figured it is probably best to pluck out just the 3 required fields to send in the PUT body